c7n/filters/core.py

```diff
@@ -27,6 +27,7 @@
 
 from dateutil.tz import tzutc
 from dateutil.parser import parse
+from pkg_resources import parse_version
```

Comment:
so rather than using pkg_resources, can we switch to distutils.version

Reply:
Updated :D I had to subclass it, since their built in version has issues.